src/app/components/core/developer/additional-services/additional-services-utils.service.ts

```diff
@@ -90,8 +90,6 @@ export class AdditionalServicesUtils {
             servicesRegistry = data.services.filter((s => (s.type.toLowerCase() === from)));
           }
           this.servicesRegistry = servicesRegistry;
-
-          this.sharedService.allServicesInfo.next(this.servicesRegistry);
```

Comment:
Why is this being removed?

Reply:
We were not using value of `allServicesInfo` anywhere, so I removed it.